<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_sync">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Synchronize Chaos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_encrypt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_decrypt">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_credits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Credits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_howto">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>How To</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_viewplots">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View Plots</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../GUI-build-desktop-Desktop_Qt_4_8_1_for_GCC__Qt_SDK__Release/Resources/home_img.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string> HYBRID CRYPTOSYSTEM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MULTICHAOTIC</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
